angularjs-1.5-basics/gulp-tasks/web.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var gulp = require('gulp');
+var $ = require('gulp-load-plugins')();
+var argv = require('yargs').argv;
```

Comment:
__[jshint]__
* 'require' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->